k8s/apis/k8s/v1alpha1/vineyardd_types.go

```diff
@@ -67,23 +67,10 @@ type ServiceConfig struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:=9600
 	Port int `json:"port,omitempty"`
-
-	// service label selector
-	// +kubebuilder:validation:Optional
-	// +kubebuilder:default:="rpc.vineyardd.v6d.io/rpc=vineyard-rpc"
-	Selector string `json:"selector,omitempty"`
```

Comment:
We need the selector here to specific a vineyard cluster as there may be several vineyard clusters.